reliabot/reliabot.py

```diff
@@ -41,10 +41,23 @@
 from typing import TextIO
 from typing import Union
 
-from ruamel.yaml import YAML  # ruamel.yaml preserves comments, PyYAML doesn't.
-from ruamel.yaml.comments import CommentedMap
-from ruamel.yaml.comments import CommentedSeq
-from ruamel.yaml.parser import ParserError
+try:
+    # ruamel.yaml preserves comments, PyYAML doesn't.
+    from ruamel.yaml import YAML
+    from ruamel.yaml.comments import CommentedMap
+    from ruamel.yaml.comments import CommentedSeq
+    from ruamel.yaml.parser import ParserError
+except ModuleNotFoundError as module_not_found:
+    error_message = "".join(
+        [
+            f"{module_not_found}",
+            "\n\n",
+            "Reliabot requires the ruamel.yaml module to preserve comments in dependabot.yml files.",
+            "\n",
+            "See https://github.com/dupuy/reliabot/#installation for installation instructions.",
+        ]
```

Comment:
To avoid complaints about lowercase for constants (since this is outside any function, as we need to do the import at global level), you could define a proper constant (`RUAMEL_NOT_FOUND`?) before `COMMENT_PREFIX` below, and move the `ruamel.yaml` imports below that (since it needs to be after `error()` anyhow).

Reply:
Changed this to match the formatting you've provided in #20